ros2param/ros2param/verb/dump.py

```diff
@@ -80,7 +67,8 @@ def main(self, *, args):  # noqa: D102
         with DirectNode(args) as node:
             yaml_output = {node_name.full_name: {'ros__parameters': {}}}
 
-            # retrieve values
+            # retrieve parameter names
+            response = None
```

Comment:
This one shouldn't be needed; we are assigning it to something in the next call.

```suggestion
```

Reply:
fixed in https://github.com/ros2/ros2cli/pull/933/commits/5b47d97452e60f7ce8fcd9abce2d8c0fd58ad277